contracts/src/interfaces/lseth/ILsETHHyperdriveRead.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { IHyperdriveRead } from "../IHyperdriveRead.sol";
+import { IRiverV1 } from "./IRiverV1.sol";
+
+interface ILsETHHyperdriveRead is IHyperdriveRead {
```

Comment:
Same here, we should add a `ILsETHHyperdrive` interface that consumes this. Check out `IStETHHyperdrive` for an example.